client/components/first-view/index.jsx

```diff
@@ -43,7 +43,9 @@ const FirstView = React.createClass( {
 	},
 
 	componentWillUnmount() {
-		this.updateDocumentStylesForHiddenFirstView();
+		process.nextTick( () => {
```

Comment:
It feels strange to use `process` in a component... Can we just use `setTimeout` instead? It seems to also work around the issue on my machine.